### DISCUSSION TITLE
when will the NZXT RGB & Fan Controller (3+6 channels) get full support?

### DISCUSSION THREAD

**Discussion by KH310DA** [liquidctl/liquidctl#563] [Q&A]
hey,

when will the NZXT 3+6 channels RGB & fan controller get full support? is it possible that this will get full support in the next few months, or is something different with this controller? I want to buy 2 of these controllers, but don't want to use NZXT CAM to adjust RGB lighting. 😅

thanks in advance for any help.

**Comment by jonasmalacofilho** [comment]
> when will the NZXT 3+6 channels RGB & fan controller get full support?

No idea, sorry.

> is it possible that this will get full support in the next few months,

Well, it's probably possible _if someone finds time to work on it._

(And in the near future it's unlikely that I'll have time/energy to do that myself).

> or is something different with this controller?

Yeah, it seems to follow a similar yet slightly different protocol than the previous 3+2-channel variants. See #541 for more information.
